source/Willow/LoadingNotificationCommand.class.st

```diff
@@ -11,32 +11,39 @@ Class {
 	#category : #'Willow-WebInteraction'
 }
 
-{ #category : #'instance creation - private' }
-LoadingNotificationCommand class >> executing: aServerIndependentActionCollection then: aModelLoadingActionCollection [
-
-	^self new initializeExecuting: aServerIndependentActionCollection
-		then: aModelLoadingActionCollection
-]
-
 { #category : #'instance creation' }
-LoadingNotificationCommand class >> onPageTopWithClasses: aCssClassCollection [
+LoadingNotificationCommand class >> displaying: aWebView applying: aComponentCommand [
 
 	^ self
 		executing:
 			(Array
 				with: [ :aScript :aCanvas | 
 					aScript << (aCanvas jQuery id: Constants >> #willow >> #notificationSectionName)
 						html: [ :aSubCanvas | 
-							aSubCanvas div
-								classes: aCssClassCollection;
-								with: 'Loading...' ] ])
+							aSubCanvas render:
+								(GenericContainerWebView wrapping: aWebView applying: aComponentCommand) ] ])
 		then:
 			(Array
 				with: [ :aScript :aCanvas | 
 					aScript << (aCanvas jQuery id: Constants >> #willow >> #notificationSectionName)
 						html: [ :aSubCanvas |  ] ])
 ]
 
+{ #category : #'instance creation - private' }
+LoadingNotificationCommand class >> executing: aServerIndependentActionCollection then: aModelLoadingActionCollection [
+
+	^self new initializeExecuting: aServerIndependentActionCollection
+		then: aModelLoadingActionCollection
+]
+
+{ #category : #'instance creation' }
+LoadingNotificationCommand class >> onPageTopWithClasses: aCssClassCollection [
+
+	^ self
+		displaying: 'Loading...'
+		applying: [:div | div addClass commandStyledWithAll: aCssClassCollection]
```

Comment:
This is not Ok. Because passing the style against the command will try to use the namespace to revolve the names, and it this case the aCssClassCollection already are the class names.